### PR TITLE
Added the Loadbalancer service for Grafana

### PR DESCRIPTION
**Pull request by Ganesh647**
Issue #, if available:

Description of changes:
Added  load balancer service for accessing the grafana dashboard 
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: substrate/monitoring/grafanalbservice.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+spec:
+  ports:
+    - port: 80
+      protocol: TCP
+      targetPort: grafana
+  selector:
+    app.kubernetes.io/name: grafana
+  sessionAffinity: None
+  type: LoadBalancer
+
```
Comment (prateekgogia) [inline substrate/monitoring/grafanalbservice.yaml:19]: remove extra line

File: substrate/README.md [hunk 2]
```diff
@@ -81,6 +81,10 @@ kubectl label configmap master-dashboard -n monitoring  grafana_dashboard=1
 
 > Note: Coming soon, auto-load these dashboards when a KIT environment is created
 
+Accessing the grafana dashbaord externally using load balancer from [monitoring](monitoring/)
+```bash
+kubectl apply -f grafanalbservice.yaml
```
Comment (prateekgogia) [inline substrate/README.md:86]:
```suggestion
[Optional] To access Grafana Dashboard externally using an NLB
```bash
kubectl apply -f monitoring/grafanalbservice.yaml
```

**Review by prateekgogia — APPROVED** [review]

**Comment by yzaccc** [comment]
please squash your commits before merge.

**Comment by hakuna-matatah** [comment]
> please squash your commits before merge.

Git does that automatically when you merge from console.

**Pull request merged**